src/AppInstallerCLITests/YamlManifest.cpp

```diff
@@ -342,36 +342,31 @@ TEST_CASE("ComplexSystemReference", "[ManifestValidation]")
 {
     Manifest manifest = YamlParser::CreateFromPath(TestDataFile("Manifest-Good-SystemReferenceComplex.yaml"));
 
-    REQUIRE(manifest.Installers.size() == 5);
-
-    // Zip installer does not inherit
-    REQUIRE(manifest.Installers[0].BaseInstallerType == InstallerTypeEnum::Zip);
-    REQUIRE(manifest.Installers[0].PackageFamilyName == "");
-    REQUIRE(manifest.Installers[0].ProductCode == "");
+    REQUIRE(manifest.Installers.size() == 4);
 
     // MSIX installer does inherit
-    REQUIRE(manifest.Installers[1].BaseInstallerType == InstallerTypeEnum::Msix);

```

Comment:
It's probably not worth changing, but we could do something like
```
auto& installer = manifest.Installer[0];
REQUIRE(installer.BaseInstallerType == InstallerTypeEnum::Msix);
...
```
so that we don't have to change the index in all places. Though I doubt we'll have to do it again...

Reply:
Changed to suggested so that we never have to think about this again :)